README.md

```diff
@@ -2,18 +2,22 @@ More info can be found on the [resin-blog](https://resin.io/blog/prometheusv2/)
 
 ### Required Environment variables
 
-| Key                | Description                           |
-|--------------------|---------------------------------------|
-| GMAIL_ACCOUNT     | Your Gmail email                      |
-| GMAIL_AUTH_TOKEN | Your Gmail password or auth token      |
-| RESIN_EMAIL       | Your resin.io email                   |
-| RESIN_PASS        | Your resin.io password                |
-| RESIN_APP_NAME   | Resin application you wish to monitor |
+| Key                | Description                                   |
+|--------------------|-----------------------------------------------|
+| GMAIL_ACCOUNT      | Your Gmail email                              |
+| GMAIL_AUTH_TOKEN   | Your Gmail password or auth token             |
+| RESIN_TOKEN        | Your resin.io token                           |
+| RESIN_EMAIL        | Your resin.io email, if not using a token     |
+| RESIN_PASS         | Your resin.io password, if not using a token  |
+| RESIN_APP_NAME     | Resin application name you wish to monitor    |
+
+These can be set in the Dockerfile, passed to Docker with `--env` at runtime, or placed in the `.env` in the root of this project in
+as `source`-able bash commands (`export RESIN_TOKEN=...`, newline separated)
```

Comment:
These should be on one line, 'These can be set in the Dockerfile, passed to Docker with `--env` at runtime, or placed in a `.env` file at the root of this project as `source`-able bash commands (`export RESIN_TOKEN=...`, newline separated).'